driver-core/src/test/functional/com/mongodb/client/CommandMonitoringTestHelper.java

```diff
@@ -253,11 +253,43 @@ private static void massageActualCommand(final BsonDocument command, final BsonD
                 command.remove(key);
             } else if (command.isDocument(key) && expectedCommand.isDocument(key)) {
                 massageActualCommand(command.getDocument(key), expectedCommand.getDocument(key));
+            } else if (command.containsKey("pipeline") && expectedCommand.containsKey("pipeline")) {
+                massagePipeline(command, expectedCommand);
             }
         }
 
     }
 
+    // SPEC-1350 introduced a change that made the fullDocument field optional. In some change stream tests, the
```

Comment:
Could we just change ChangeStreamOperation instead.  So instead of:

    BsonDocument changeStream = new BsonDocument("fullDocument", new BsonString(fullDocument.getValue()));

we do

     BsonDocument changeStream = new BsonDocument();
     if (fullDocument != FullDocument.DEFAULT) {
             changeStream.append("fullDocument", new BsonString(fullDocument.getValue()));
     }